tests/test_diff.py

```diff
@@ -1121,7 +1130,7 @@ def _extract(diff_json):
         # changes from HEAD (R1 -> WC)
```

Comment:
comments to remove?

Reply:
seems like a fairly useful part of the test AFAICT. the test passes when it's uncommented. I'll uncomment it.